src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/ResetNullEventTimestampMessageJobTest.java

```diff
@@ -67,87 +75,137 @@ void should_be_able_to_run_reset_null_event_timestamp_problem_message_job() {
+                                                         null,
+                                                         objectMapper
+        ).run().isEmpty());
+        assertTrue(new ResetNullEventTimestampMessageJob(caseEventMessageRepository,
+                                                         List.of(),
+                                                         objectMapper
+        ).run().isEmpty());
     }
 
     @Test
     void should_return_empty_response_for_empty_unprocessable_messages() {
         when(caseEventMessageRepository.findByMessageId(messageIds)).thenReturn(List.of());
         assertTrue(resetNullEventTimestampProblemMessageJob.run().isEmpty());
 
-        when(caseEventMessageRepository.findByMessageId(messageIds))
-            .thenReturn(List.of(buildMessageEntity("messageId_1", MessageState.NEW),
-                                buildMessageEntity("messageId_2", MessageState.READY)));
+        caseEventMessageEntityMap.put("messageId", "messageId_1");
+        CaseEventMessageEntity entityOne = buildMessageEntity(caseEventMessageEntityMap, MessageState.NEW);
+
+        caseEventMessageEntityMap.put("messageId", "messageId_2");
+        CaseEventMessageEntity entityTwo = buildMessageEntity(caseEventMessageEntityMap, MessageState.READY);
+
+        when(caseEventMessageRepository.findByMessageId(messageIds)).thenReturn(List.of(entityOne, entityTwo));
+
         assertTrue(resetNullEventTimestampProblemMessageJob.run().isEmpty());
     }
 
     @Test
     void should_return_message_id_list_response_for_handling_null_event_timestampe_messages()
         throws JsonProcessingException {
-        EventInformation eventMessageFromEntity = getEventInformation();
+        EventInformation eventMessageFromEntity = getEventInformation_one();
+
+        caseEventMessageEntityMap.put("messageId", "messageId_3");
+        CaseEventMessageEntity nullEventTimestampEntity = buildMessageEntity(caseEventMessageEntityMap,
+                                                                             MessageState.UNPROCESSABLE
+        );
 
-        CaseEventMessageEntity nullEventTimestampEntity = buildMessageEntity("messageId_3", MessageState.UNPROCESSABLE);
         nullEventTimestampEntity.setMessageContent(eventMessageFromEntity.toString());
 
-        when(caseEventMessageRepository.findByMessageId(messageIds))
-            .thenReturn(List.of(nullEventTimestampEntity));
-        when(objectMapper.readValue(nullEventTimestampEntity.getMessageContent(), EventInformation.class))
-            .thenReturn(eventMessageFromEntity);
+        when(caseEventMessageRepository.findByMessageId(messageIds)).thenReturn(List.of(nullEventTimestampEntity));
 
-        assertEquals(messageIds,resetNullEventTimestampProblemMessageJob.run());
+        when(objectMapper.readValue(nullEventTimestampEntity.getMessageContent(), EventInformation.class)).thenReturn(
+            eventMessageFromEntity);
```

Comment:
Seems to be a lot of format changes.  Can you check with the team that you and the rest of the PET team have the correct format for code?